fingerprint/src/main/java/org/openmrs/module/muzimabiometrics/EnrollFingerprint.java

```diff
@@ -0,0 +1,183 @@
+                    request.setHeader("Content-type", "application/octet-stream");
+                    HttpResponse response = client.execute(request);
+                    BufferedReader rd = new BufferedReader(new InputStreamReader(response.getEntity().getContent()));
+                    String line = "";
+                    while ((line = rd.readLine()) != null){
+                        System.out.println(line);
+                    }
+                    enrollFingerSecondTime();
+                    statusLabel.setText("First Capturing Completed");
+                } catch (JlibFprint.EnrollException e) {
+                    e.printStackTrace();
+                }
+                catch(IOException ex){
+                }
+            }
+        });
+        controlPanel.add(firstCaptureBtn);
+        mainFrame.setVisible(true);
+    }
+    private void enrollFingerSecondTime(){
+        firstCaptureBtn.setVisible(false);
+        secondCaptureBtn = new JButton("Capture Thumb Finger Second Time");
+        secondCaptureBtn.addActionListener(new ActionListener() {
+            public void actionPerformed(ActionEvent actionEvent) {
+                try {
+                    secondImage = jlibfprint.enroll_finger();
+                    CloseableHttpClient client = HttpClientBuilder.create().build();
+                    ByteArrayOutputStream secondFingerBO = new ByteArrayOutputStream();
+                    ObjectOutputStream oos = new ObjectOutputStream(secondFingerBO);
+                    oos.writeObject(secondImage);
+                    oos.flush();
+                    byte[] secondFingerEncodedByteArray = Base64.encodeBase64(secondFingerBO.toByteArray());
+                    System.out.println("encodedByteArray +++++++++++++++++++"+secondFingerEncodedByteArray);
+                    String url="http://localhost:8080/openmrs/module/muzimabiometrics/enrollSecondImage.form";
+                    HttpPost request=new HttpPost(url);
+                    request.setEntity(new ByteArrayEntity(secondFingerEncodedByteArray));
+                    request.setHeader("Content-type", "application/octet-stream");
+                    HttpResponse response = client.execute(request);
+                    BufferedReader rd = new BufferedReader(new InputStreamReader(response.getEntity().getContent()));
+                    String line = "";
+                    while ((line = rd.readLine()) != null) {
+                        System.out.println(line);
+                    }
+                    enrollFingerThirdTime();
+                    statusLabel.setText("Second Capturing Completed");
+                } catch (JlibFprint.EnrollException e) {
+                    e.printStackTrace();
+                }
+                catch(IOException ex){
+                }
+            }
+        });
+        controlPanel.add(secondCaptureBtn);
+        mainFrame.setVisible(true);
+    }
+    private void enrollFingerThirdTime(){
```

Comment:
I think the three enrollFinger methods could be refactored for reusability since they all do the same thing.